catalog.html

```diff
@@ -1,143 +1,166 @@
+            <div class="template-card-wrapper">
+              <h3 class="template-card-title">
+                <a href="#">A PLUS</a>
+              </h3>
+              <p>Лендинг курсов повышения квалификации</p>
+              <button class="button template-card-button" type="button">9 900 ₽</button>
+            </div>
+          </article>
         </li>
         <li>
-          <img src="#" alt="Крафтовый квас &#34;Кваст&#34;" />
-          <div>
-            <a href="#">КВАСТ</a>
-            <p>Промо-сайт производителя крафтового кваса</p>
-            <button type="button">9 900 ₽</button>
-          </div>
+          <article class="template-card">
+            <img src="#" alt="Крафтовый квас &#34;Кваст&#34;" />
+            <div class="template-card-wrapper">
+              <h3 class="template-card-title">
+                <a href="#">КВАСТ</a>
+              </h3>
+              <p>Промо-сайт производителя крафтового кваса</p>
+              <button class="button template-card-button" type="button">9 900 ₽</button>
+            </div>
+          </article>
         </li>
       </ul>
       <!--погинация-->
-      <ul>
+      <ul class="catalog-pagination-list list-reset">
         <li>
-          <a>1</a>
+          <a class="button">1</a>
         </li>
         <li>
-          <a href="#">2</a>
+          <a class="button" href="#">2</a>
         </li>
         <li>
-          <a href="#">3</a>
+          <a class="button" href="#">3</a>
         </li>
         <li>
-          <a href="#">следующая</a>
+          <a class="button" href="#">следующая</a>
         </li>
       </ul>
     </section>
   </main>
-  <footer>
-    <section>
-      <h2>Адрес студии</h2>
-      <div>
-        <p>NЁRDS DESIGN STUDIO</p>
-        <p>
-          191186, Санкт-Петербург,<br>
-          ул. Б. Конюшенная, д. 19/8
-        </p>
-        <p>тел. <a href="tel:+78122757575">+7 (812) 275-75-75</a></p>
-        <a href="#">Напишите нам</a>
-        <!--Тут будет карта-->
-      </div>
-    </section>
-    <section>
-      <h2>Давайте дружить</h2>
-      <ul>
-        <li>
-          <a href="#">Мы Вконтакте</a>
-        </li>
-        <li>
-          <a href="#">Мы в Фейсбук</a>
-        </li>
-        <li>
-          <a href="#">Мы в инстаграм</a>
-        </li>
-      </ul>
-      <p>Давайте дружить, это выгодно</p>
-      <p>Скидка 10% для друзей из социальных сетей.</p>
-    </section>
+  <footer class="footer">
+    <h2>Адрес студии</h2>
+    <!--Тут будет карта-->
+    <div class="footer-contacts">
+      <h2>NЁRDS DESIGN STUDIO</h2>
+      <p>
+        191186, Санкт-Петербург,<br>
+        ул. Б. Конюшенная, д. 19/8
+      </p>
+      <p>тел. <a class="footer-phone" href="tel:+78122757575">+7 (812) 275-75-75</a></p>
+      <a class="button footer-contacts-button" href="#">Напишите нам</a>
+    </div>
+    <ul class="footer-social-list list-reset">
+      <li>
+        <a class="button footer-social-button" href="#">Мы Вконтакте</a>
```

Comment:
Соц. сетям не нужен класс button, так как стили у ссылок уникальные и к общим кнопкам не относятся
```suggestion
        <a class="footer-social-button" href="#">Мы Вконтакте</a>
```

Reply:
убрал